icoscp/stilt/stiltobj.py

```diff
@@ -421,15 +426,22 @@ def __columns(self, cols):
             columns = ('["isodate", "latstart", "lonstart"]')
 
         elif cols=='all':
-            columns = ('["isodate","co2.stilt","co2.fuel","co2.bio","co2.fuel.coal",'+
-                    '"co2.fuel.oil","co2.fuel.gas","co2.fuel.bio","co2.energy",'+
-                    '"co2.transport", "co2.industry","co2.others", "co2.cement",'+
-                    '"co2.background", "co.stilt","co.fuel","co.bio","co2.bio.gee",' +
-                    '"co2.bio.resp", "co.fuel.coal",'+
-                    '"co.fuel.oil","co.fuel.gas","co.fuel.bio","co.energy",'+
-                    '"co.transport","co.industry","co.others", "co.cement",'+
-                    '"co.background","rn", "rn.era","rn.noah","wind.dir",'+
-                    '"wind.u","wind.v","latstart","lonstart"]')
+            columns = ('["isodate","co2.stilt","co2.bio","co2.fuel","co2.cement",'+
+                       '"co2.bio.gee", "co2.bio.resp",' +
+                       '"co2.fuel.coal","co2.fuel.oil","co2.fuel.gas",'+
+                       '"co2.fuel.bio","co2.fuel.waste",'+
+                       '"co2.energy","co2.transport", "co2.industry",'+
+                       '"co2.residential","co2.other_categories",'+
+                       '"co2.background",'+
+                       '"co.stilt","co.fuel","co.cement",'+
+                       '"co.fuel.coal","co.fuel.oil","co.fuel.gas",'+
+                       '"co.fuel.bio","co.fuel.waste",'+
```

Comment:
The component `co.fuel.waste` is not mentioned in [pylib_stilt_order.txt](https://github.com/ICOS-Carbon-Portal/pylib/files/8540455/pylib_stilt_order.txt) in `all columns`.  Is it missing from the text file or should we remove it here?

Reply:
I forgot to add co.fuel.waste in `all columns` in my text file, so please keep it in the code. 

The description in `docs/modules.md` should be adjusted as well, i.e. add new columns, change order.